core/modes/modes.py

```diff
@@ -12,6 +14,27 @@
 for key, value in modes.items():
     mod.mode(key, value)
 
+ctx_sleep.matches = r"""
+mode: sleep
+"""
+
+ctx_awake.matches = r"""
+mode: command
+mode: dictation
```

Comment:
not sure it's better, but could we do `not mode: sleep` instead? Also, I wonder if there's a context that more directly corresponds to speech being enabled / disabled 🤔

Reply:
We could add a custom scope that simply indicates whether or not speech is enabled?

Reply:
Is that possible?

Reply:
Looking into this, it appears it is not possible to do a custom scope for this. In particular, I'm not sure when to call scope.update.

It probably makes sense as a future request to add to the existing speech scope. 

As long as there are no objections to not mode: sleep, I'll move forward with that